asciidoc-confluence-publisher-maven-plugin/src/it/java/org/sahli/asciidoc/confluence/publisher/maven/plugin/testutils/ConfluenceServerSetup.java

```diff
@@ -0,0 +1,35 @@
+package org.sahli.asciidoc.confluence.publisher.maven.plugin.testutils;
+
+import org.testcontainers.Testcontainers;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+
+import java.time.Duration;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+public class ConfluenceServerSetup {
+
+    private static final int PORT = 8090;
+
+    public static ConfluenceServer setupContainer() {
+        GenericContainer genericContainer = new GenericContainer("confluencepublisher/confluence-publisher-it:6.0.5")
+                .withExposedPorts(PORT)
+                .withReuse(true)
```

Comment:
When running locally you need to add to `~/.testcontainers.properties` file line:
```
testcontainers.reuse.enable=true
```
It will make Testcontainers to not shutdown a test container after the tests, so when debugging/fixing tests you won't need to wait for the Confluence server startup.

Here is more documentation on this feature: https://github.com/cwansart/testcontainers-java/blob/a4a7e2d60a048112cc4148961edeb4dfdd6909f9/docs/features/reusable_containers.md